src/schematools/utils.py

```diff
@@ -59,7 +64,12 @@ def profile_defs_from_url(profiles_url: Union[URL, str]) -> Dict[str, types.Prof
 
     The URL could be ``https://schemas.data.amsterdam.nl/profiles/``
     """
-    return defs_from_url(base_url=profiles_url, data_type=types.ProfileSchema)
+    return {
+        p_name: p_schema
+        for p_name, p_schema in defs_from_url(
+            base_url=profiles_url, data_type=types.ProfileSchema
+        ).items()
+    }
```

Comment:
Hebben we dit echt nodig?

Reply:
Kon gelukkig weer verwijderd worden agv. fix door @gkoller bij het gebruik van `ST` typevar.